### PR TITLE
ref(search): Be consistent about allowed spaces in lists

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Allow spaces before commas

### CODE REVIEW

File: src/sentry/api/event_search.py [hunk 1]
```diff
@@ -622,9 +622,7 @@ def visit_text_filter(self, node, children):
     def visit_text_in_filter(self, node, children):
         (negation, search_key, _, search_value) = children
         operator = "IN"
-        search_value = SearchValue(
-            self.process_list(search_value[1], [(_, _, val) for _, _, val in search_value[2]])
```
Comment (EvanPurkhiser) [inline src/sentry/api/event_search.py:626]: This might have been from when there was an extra token after? not sure

Comment (wedamija) [reply src/sentry/api/event_search.py:626]: Yeah probably a holdover from before your other refactors... a bit surprised that both work though

**Review by wedamija — APPROVED** [review]

File: src/sentry/api/event_search.py (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/api/event_search.py:626]: The list generation here was completely a no-op

Comment (wedamija) [reply src/sentry/api/event_search.py:626]: Lol, wtf was I doing... I wonder if I was unpacking at some point

**Pull request merged**